get-started-building-a-blitz-website-with-directus/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Get Started Building a Blitz Website with Directus
+date: 2022-02-07
```

Comment:
May need to update this later on ([reference](https://github.com/directus/guides/commit/c842cc7d14becb465414ba1035699799c369ea06)), just a note 👍